packages/components/src/styles/colors.ts

```diff
@@ -94,7 +94,7 @@ export const colors = {
   fillTransparent200: '#00000026',
   fillTransparent100: '#00000000',
   fillOverlay: '#00000066',
-  fillStatePrimaryEnabled: '#dd0303ff',
+  fillStatePrimaryEnabled: '#414141',
```

Comment:
double checking: is this correct?

Reply:
Yes, I double checked DSM and state primary enabled is #414141 not #dd0303ff